examples/hashsmt/sha1_inversion.cpp

```diff
@@ -54,19 +54,14 @@ int main(int argc, char* argv[]) {
     ofstream output(argv[2]);
     output << expr::ExprSetDepth(-1) << language::SetLanguage(language::output::LANG_SMTLIB_V2);
     output << SetBenchmarkLogicCommand("QF_BV") << endl;
-    output << SetBenchmarkStatusCommand(SMT_SATISFIABLE) << endl;
+
+    hashsmt::Word message(msg.size()*8, "x");
```

Comment:
If I am reading this correctly, then `msg` is now unused and it is just always all `x`s of the same size as `argv[1]`. Is this intentional?